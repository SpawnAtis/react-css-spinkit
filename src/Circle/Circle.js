import React from 'react'
import styles from './styles.css'

const Circle = () => {
  return (
    <>
      <style type="text/css">{`${styles}`}</style>
      <div className="lds-circle">
        <div />
      </div>
    </>
  )
}

export default Circle
