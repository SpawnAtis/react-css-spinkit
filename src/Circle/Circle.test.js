import React from 'react'
import {render} from 'react-testing-library'
import Circle from './Circle'

describe('Circle component', () => {
  it('Should render Circle without problems', () => {
    const {asFragment} = render(<Circle />)
    expect(asFragment()).toMatchSnapshot()
  })
})
