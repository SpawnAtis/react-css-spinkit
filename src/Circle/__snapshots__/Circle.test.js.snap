// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Circle component Should render Circle without problems 1`] = `
<DocumentFragment>
  <style
    type="text/css"
  >
    [object Object]
  </style>
  <div
    class="lds-circle"
  >
    <div />
  </div>
</DocumentFragment>
`;
